javascript/javascript1/week2/formal.js

```diff
@@ -0,0 +1,29 @@
+
+
+// Formal fullname 
+
+
+
+let firstName =" Benjamin"; 
+ let surName = " Hughes"; 
+ let gender= "male"; 
+ let useFormalName = true ; 
+
+  function  getFullname (firstName , surName , useFormalName, gender){
+
+    if (useFormalName === true && gender === "male" ){
+        return `lord  ${firstName} + ${surName}`;
```

Comment:
Nit: + symbol is not required here because it would print
`lord Benjamin + Huges`